msg_queue.go

```diff
@@ -28,6 +32,27 @@ func (m *msgQueue) pushMessage(message *MessageReq) {
 	heap.Push(queue, message)
 }
 
+// pushPendingCommitMessage adds a new commit message to a message queue
+func (m *msgQueue) pushPendingCommitMessage(message *MessageReq) {
+	m.queueLock.Lock()
+	defer m.queueLock.Unlock()
+
+	m.pendingCommitMsgsQueue.Push(message)
+}
+
+// processPendingCommitMessages processes accumulated pending commit messages for the given sequence and round.
+func (m *msgQueue) processPendingCommitMessages(view *View, handler func(*MessageReq)) {
```

Comment:
Why not make this `handler` a field inside `msgQueue`? It is always a reference to the same function.